doc/integrator/docker.rst

```diff
@@ -6,8 +6,15 @@ Use Docker to deploy your application
 Architecture schema
 -------------------
 
-.. image:: docker.png
-.. source file is docker.dia.
+For OpenShift projects:
+
+.. image:: docker-openshift.png
```

Comment:
In this diagram:
* "redis" -> is this not currently being replaced by something else?
* "Postgis" -> in my opinion, it would be more precise to write PostgreSQL here
* why are we underlining "S3 like"? And: how about using the term "Object storage" instead (same term as on doc page integrator/objectstorage)